bundles/org.openhab.binding.novafinedust/README.md

```diff
@@ -0,0 +1,78 @@
+
+### Polling
+
+If one needs data in different intervals, i.e. not as fast as possible and not in intervals that are a multiple of full minutes, polling can be configured. The `pollingInterval` parameter specifies the time in seconds when data will be polled from the sensor.
+
+In addition to the mode one has to provide the port to which the device is connected.
+
+A full overview about the parameters of the `SDS011` thing is given in the following table:
+
+| parameter name    | mandatory | description                                                                           |
+|-------------------|-----------|---------------------------------------------------------------------------------------|
+| port              | yes       | the port the sensor is connected to, i.e. /dev/ttyUSB0.                              |
+| reporting         | yes       | whether the reporting mode (value=true) or polling mode should be used.               |
+| reportingInterval | no        | the time in minutes between reportings from the sensor (default=1, min=0, max=30).    |
+| pollingInterval   | no        | the time in seconds between data polls from the device. (default=10, min=3, max=3600) |
+
+## Channels
+
+Since the supported device is a sensor, both channels are read-only channels. 
+
+| channel  | type           | description                   |
+|----------|----------------|-------------------------------|
+| pm25     | Number:Density | This provides the PM2.5 value |
+| pm10     | Number:Density | This provides the PM10 value  |
+
+## Full Example
+
+demo.things:
+
+```
+Thing novafinedust:SDS011:mySDS011Report "My SDS011 Fine Dust Sensor with reporting" [ port="/dev/ttyUSB0", reporting=true, reportingInterval=1 ]
+Thing novafinedust:SDS011:mySDS011Poll "My SDS011 Fine Dust Sensor with polling" [ port="/dev/ttyUSB0", reporting=false, pollingInterval=10 ]
+```
+
+demo.items:
+
+```
+Number:Density PM25 "My PM 2.5 value" { channel="novafinedust:SDS011:mySDS011Report:pm25" }
+Number:Density PM10 "My PM 10 value" { channel="novafinedust:SDS011:mySDS011Report:pm10" }
+```
+
+demo.sitemap:
+
+```
+sitemap demo label="Main Menu"
+{
+    Frame {
+        Text item=PM25 label="My PM 2.5 value"
+        Text item=PM10 label="My PM 10 value"
+    }
+}
+```
+
+## Limitations
+
+In theory once can have multiple sensors connected and distinguish them via their device ID. However, this is currently not implemented and the binding always configures any device and accepts data reportings from any device too.
```

Comment:
You mean there could be multiple sensors on the same port, right?
I very much hope that it is possible to have multiple `SDS011` things that use different ports.
Maybe you can clarify this in the statement here a bit.